config/settings.yml

```diff
@@ -9,7 +9,7 @@ defaults: &defaults
   site_terms: ''
   site_contact_username: ''
   site_contact_email: ''
-  registrations_mode: 'open'
+  registrations_mode: 'none'
```

Comment:
I thought "none" was no one could sign up by default? Wasn't the idea to use approvals by default?

Reply:
hmm, maybe we add a little JavaScript so when you select "open" we show a yellow warning message saying "We recommend using approvals only, unless you've a 24x7 moderation team in place, since open registrations can be abused for spam and malicious activity" or something? 

Like really drive them towards selecting "approvals" over "open" then.

Reply:
![image](https://github.com/mastodon/mastodon/assets/384364/eafbe499-4243-4e62-a86f-f78c73eae89e)
![image](https://github.com/mastodon/mastodon/assets/384364/0b19424b-199e-4d03-ab7b-26540b843ea8)

Reply:
Maybe drop require a reason to a new row? Looks kinda out of place as a floating column

Reply:
Agreed, this will probably look better

Reply:
I like the "Approval-only by default" approach, and setting default config to "none" would basically make most servers inaccessible.

Reply:
Taking a look at this suggestion, I'm not sure this really is an improvement. I have a slight preference to go with the least amount of changes from the existing layout, but I'm fine going either way.

## Before

![image](https://github.com/mastodon/mastodon/assets/384364/e4eb26e3-2457-4006-a13b-6460628ce3dd)
![image](https://github.com/mastodon/mastodon/assets/384364/fb7ffbab-ca14-4970-a4e0-f6047f090cbd)

## After

![image](https://github.com/mastodon/mastodon/assets/384364/9fccbb45-13c0-4e79-8eda-6378f93f42b9)
![image](https://github.com/mastodon/mastodon/assets/384364/1ae21fba-33c9-4576-8ff3-4ea1421414ef)

Reply:
Those screen will need a redesign some day, and are only seen by admins. Let's go with the current version.